packages/core/src/internal/services/focus-trap-tracker.service.spec.ts

```diff
@@ -6,220 +6,160 @@
 
 import { html } from 'lit';
 import { createTestElement, removeTestElement } from '@cds/core/test';
-import {
-  CDS_FOCUS_TRAP_DOCUMENT_ATTR,
-  CDS_FOCUS_TRAP_ID_ATTR,
-  FocusTrapTracker,
-} from './focus-trap-tracker.service.js';
+import { FocusTrapTrackerService } from './focus-trap-tracker.service.js';
 import { arrayTail } from '../utils/array.js';
 
 describe('Focus Trap Tracker Service - ', () => {
-  const service = FocusTrapTracker;
-  const testTrapIds = ['abcd', 'efg', 'hijk', 'lmnop'];
+  const testTrapElements = [
+    { focusTrapId: 'abcd' },
+    { focusTrapId: 'efg' },
+    { focusTrapId: 'hijk' },
+    { focusTrapId: 'lmnop' },
+  ];
 
   afterEach(() => {
-    // reset
-    service.getDocroot().removeAttribute(CDS_FOCUS_TRAP_DOCUMENT_ATTR);
+    FocusTrapTrackerService.setTrapElements([]);
   });
 
   describe('getDocroot(): ', () => {
     it('should return something', () => {
-      const myDocroot: HTMLElement = service.getDocroot();
-      expect(myDocroot).not.toBeNull('Docroot node should exist');
-      expect(myDocroot.tagName.toLowerCase()).toBe('html', 'should return the documentElement(html) node');
+      expect(document.documentElement).not.toBeNull('Docroot node should exist');
+      expect(document.documentElement.tagName.toLowerCase()).toBe(
+        'html',
+        'should return the documentElement(html) node'
+      );
```

Comment:
todo: remove test, I think this can be removed since we no longer track on the doc root but in the state object